internal/commands/list.go

```diff
@@ -0,0 +1,96 @@
+package commands
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	"github.com/deislabs/duffle/pkg/claim"
+
+	"github.com/docker/app/internal/store"
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/docker/cli/cli/config"
+	units "github.com/docker/go-units"
+	"github.com/spf13/cobra"
+)
+
+type listOptions struct {
+	targetContext string
+}
+
+var (
+	listColumns = []struct {
+		header string
+		value  func(c *claim.Claim) string
+	}{
+		{"INSTALLATION", func(c *claim.Claim) string { return c.Name }},
+		{"ACTION", func(c *claim.Claim) string { return c.Result.Action }},
+		{"RESULT", func(c *claim.Claim) string { return c.Result.Status }},
+		{"CREATED", func(c *claim.Claim) string { return units.HumanDuration(time.Since(c.Created)) }},
+		{"MODIFIED", func(c *claim.Claim) string { return units.HumanDuration(time.Since(c.Modified)) }}}
+)
```

Comment:
Should we add a column `BUNDLE REFERENCE` (with the value being either the registry reference, or the path to the app definition, depending on the installation having been processed from a registry reference or a path to a definition / bundle.json on disk) ?

Reply:
I don't think the path to the definition is a good information (the store is obscure with digested directories), I don't think we want to expose that.
But maybe we can add a column with the bundle name and its version, which can be useful with multiple instances of the same application.

Reply:
For apps deployed from a registry reference, I'd love to see it in this list